hackathon_site/event/jinja2/event/landing.html

```diff
@@ -325,7 +325,11 @@ <h3 class="center white-text">Sponsors</h3>
             </p>
         </div>
     </div>
+    <div class="container section scrollspy" id="past-projects">
```

Comment:
Could you increase the space between this past projects text and the carousel

Reply:
added 40px bottom padding